packages/shared/components/organisms/NetworkSideDrawer.svelte

```diff
@@ -0,0 +1,23 @@
+<script lang="ts">
+    import { localize } from '@core/i18n'
+    import { isNetworkSideDrawerOpen } from '@core/network/stores'
+    import { SideDrawer, Text, TextType, Icon } from 'shared/components/'
```

Comment:
We should be able to use the `@ui` path alias here, although because may not work because this file is also within the `shared/components` folder.